tools/build/rules/grpc_c++.bzl

```diff
@@ -0,0 +1,250 @@
+# Copyright (C) 2018 Google Inc.
```

Comment:
@pmuetschard please can you review this (likely post submission) when you're back?